skipruntime-ts/ts/src/skipruntime_api.ts

```diff
@@ -274,12 +217,7 @@ export interface AsyncLazyCollection<
   M extends TJSON,
 > extends LazyCollection<K, Loadable<V, M>> {}
 
-/**
- * An _Eager_ reactive collection, whose values are computed eagerly and kept up
- * to date whenever inputs are changed
- */
-export interface EagerCollection<K extends TJSON, V extends TJSON>
-  extends Constant {
+export interface CollectionReader<K extends TJSON, V extends TJSON> {
   /**
    * Get (and potentially compute) all values mapped to by some key of a lazy reactive
```

Comment:
Unrelated to this PR but this probably means to read "some key of a ~lazy~ eager reactive collection".